docs/content/template-files.md

```diff
@@ -172,6 +172,31 @@ referenced projects into the package.
 include-referenced-projects true
 ```
 
+If referenced project has it's own template file then it will be added to package as 
```

Comment:
If *the* referenced project has *its* own ... added to *the* package as *a*

Please link "template file" like you did above.

Reply:
Not sure about link. This is the template file page already.

Reply:
Right, I missed that!